js/main.js

```diff
@@ -17,4 +44,59 @@ const getRandomInt = (min, max) => {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 };
 
-getRandomInt(2, 4);
+const getAuthorAvatar = () => {
+  const result = getRandomInt(1, 10);
+  if (result < 10) {
+    return `img/avatars/user0${result}.png`;
+  } else {
+    return `img/avatars/user${result}.png`;
+  }
```

Comment:
Два раза дублируется одна и та же информация, лучше "заранее" посчитать цифру аватара и вернуть 1 раз строку
```suggestion
  const avatarNum = res < 10 ? .... : ......
  
  return `img/avatars/user${avatarNum}.png`;
```